docker/scripts/startup.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+echo "Checking GPU presence and CUDA linking"
```

Comment:
should this also take arguments for framework, workload, submission, tuning search space, and exp name?

Reply:
cool hacked the flags through Dockerfile -> startup.sh -> git hosted startup.sh 

last file is editable by us at any time and I kept it so as rebuilding images is time consuming